setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = aiidalab_mfa_cscs
-version = 0.1-alpha
```

Comment:
The 0.1-alpha was good. You can change it to `0.1.0a1` also

Reply:
This is not an actual release, after merge I'll run bumpver to make release of `0.1.0a0`.

Reply:
Anyway I change it to `v0.1.0a0` ;)

Reply:
You can release `v0.1.0` also starting from the alpha release.

Also, we don't need to release alpha's. If the version is `0.x.x` we are allowed to do breaking changes. This is what the `semantic` version says.

Which brings me to another question: do we want to have it semantic? Why don't we do it the date based (same as QE app)?

Reply:
I don't know. I prefer the version of aiidalab-launch `v2023.1018` in format `version_pattern = "vYYYY.BUILD[PYTAGNUM]"`

Reply:
> I don't know. I prefer the version of aiidalab-launch `v2023.1018` in format `version_pattern = "vYYYY.BUILD[PYTAGNUM]"`

Maybe use this schema then, I am ok with that!

Reply:
Voila, it is "v2023.0000" and after run `bumpver` it will be "v2023.1001".